### ISSUE TITLE
Accessibility Checker

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#10598]
Would be cool to have an accessibility checker (maybe as analyzer plugin) that tells people what's missing in their flutter app to make it accessible. It should for example warn if there are controlls without meaningful labels.

**Comment by goderbauer** [comment]
Also warn if touch targets are too small

**Comment by eseidelGoogle** [comment]
It seems like we should (at least for now) just use the ones which exist for Android and iOS?

**Comment by goderbauer** [comment]
Yeah, I don't think this is high priority right now. However, in the future it would be nice if we could have this kind of feedback right in the IDE.

**Comment by jonahwilliams** [comment]
I've done some possible work towards this in https://github.com/flutter/flutter/pull/20462.

**Comment by zanderso** [comment]
Can this be close? Alternately does this need support from the analyzer team?

**Comment by jonahwilliams** [comment]
While the PR above allows us to check in unit tests, it doesn't have quite the same functionality as something like the Android accessibility inspector (https://developer.android.com/guide/topics/ui/accessibility/testing) which allows you to inspect attributes or contrast in a running application. Now that we have devtools, it might be time to revist integration of the two.

fyi @jacob314

**Comment by matanlurey** [comment]
Closing as not planned.
